test/connectors/BackLinkCacheConnectorSpec.scala

```diff
@@ -17,44 +17,50 @@
 package connectors
 
 import config.ApplicationConfig
-import models.{BackLinkModel, ReviewDetails}
+import models._
 import org.mockito.ArgumentMatchers.any
-import org.mockito.MockitoSugar
+import org.mockito.Mockito._
+import org.scalatest.BeforeAndAfterEach
 import org.scalatestplus.play.PlaySpec
 import org.scalatestplus.play.guice.GuiceOneServerPerSuite
 import play.api.libs.json.Json
-import play.api.test.Helpers._
-import play.api.test.Injecting
-import uk.gov.hmrc.http.cache.client.{CacheMap, SessionCache}
-import uk.gov.hmrc.http.{HeaderCarrier, SessionId}
-import uk.gov.hmrc.play.bootstrap.http.DefaultHttpClient
-import scala.concurrent.ExecutionContext
+import uk.gov.hmrc.connectors.ConnectorTest
+import uk.gov.hmrc.http._
+import uk.gov.hmrc.http.cache.client.SessionCache
+import uk.gov.hmrc.http.cache.client.CacheMap
 
-import scala.concurrent.Future
+import java.util.UUID
+import scala.concurrent.{ExecutionContext, Future}
 
-class BackLinkCacheConnectorSpec extends PlaySpec with GuiceOneServerPerSuite with MockitoSugar with Injecting {
+class BackLinkCacheConnectorSpec extends PlaySpec with GuiceOneServerPerSuite with ConnectorTest with BeforeAndAfterEach {
+
```

Comment:
ConnectorTest should be used on each test rather than the Spec class to ensure no interaction between tests

Reply:
Resolved.